examples/cpp/topic_instances/CLIParser.hpp

```diff
@@ -0,0 +1,739 @@
+        std::cout << "                                  (Default: Square) "                     << std::endl;
+        std::cout << "  -s <num>, --samples <num>       Number of samples per instance to send" << std::endl;
+        std::cout << "                                  or receive.    [0 <= <num> <= 65535]"   << std::endl;
+        std::cout << "                                  (Default: 0 [unlimited])"               << std::endl;
+        std::cout << "Publisher options:"                                                       << std::endl;
+        std::cout << "  -c <str>, --color <str>         Shape color. Possible values are: "     << std::endl;
+        std::cout << "                                   · RED         · CYAN "                 << std::endl;
+        std::cout << "                                   · BLUE        · MAGENTA "              << std::endl;
+        std::cout << "                                   · GREEN       · PURPLE "               << std::endl;
+        std::cout << "                                   · YELLOW      · GREY "                 << std::endl;
+        std::cout << "                                   · ORANGE      · BLACK "                << std::endl;
+        std::cout << "                                  (Default: first 4 colors: "             << std::endl;
+        std::cout << "                                   RED, BLUE, GREEN, and YELLOW)"         << std::endl;
+        std::cout << "            --height <num>        Shape bounds height "                   << std::endl;
+        std::cout << "                                  (Default: 265)"                         << std::endl;
+        std::cout << "            --interval <num>      Time between samples in milliseconds "  << std::endl;
+        std::cout << "                                  (Default: 100)"                         << std::endl;
+        std::cout << "            --size <num>          Shape size "                            << std::endl;
+        std::cout << "                                  [10 <= <num> <= 100]"                   << std::endl;
+        std::cout << "                                  (Default: 30)"                          << std::endl;
+        std::cout << "            --step <num>          Shape step movement "                   << std::endl;
+        std::cout << "                                  (Default: 3)"                           << std::endl;
+        std::cout << "  -t <num>, --timeout <num>       Wait time until automatically stop the" << std::endl;
+        std::cout << "                                  publisher execution once all samples "  << std::endl;
+        std::cout << "                                  have been sent, in seconds "            << std::endl;
+        std::cout << "                                  (Default: 10)"                          << std::endl;
+        std::cout << "  -w <num>, --width <num>         Shape bounds width "                    << std::endl;
```

Comment:
The short argument should always be available if possible. In those which has the same initial letter, the short option is not explicit so it is not added.

Reply:
I agree but since you couldn't add any of the other short argument for me it's better if we remove also this one to be consistent. This is in my opinion so feel free to leave it as it is if you don't agree